src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -724,34 +724,34 @@ public static partial class Buffer
         static byte System.Numerics.IMultiplicativeIdentity<System.Byte,System.Byte>.MultiplicativeIdentity { get { throw null; } }
         static byte System.Numerics.INumberBase<System.Byte>.One { get { throw null; } }
         static byte System.Numerics.INumberBase<System.Byte>.Zero { get { throw null; } }
-        public static System.Byte Clamp(System.Byte value, System.Byte min, System.Byte max) { throw null; }
-        public int CompareTo(System.Byte value) { throw null; }
+        public static byte Clamp(byte value, byte min, byte max) { throw null; }
```

Comment:
Changes were automated via https://github.com/dotnet/arcade/pull/8769, removing the need to continuously "manually" update for static abstracts